modules/sdk/project-templates/project-templates-theme-contributor/src/main/resources/archetype-resources/pom.xml

```diff
@@ -78,4 +83,12 @@
 			</plugin>
 		</plugins>
 	</build>
+	<dependencies>
+		<dependency>## ignore dependency comparison
```

Comment:
This is a phrase so it should be capitalized - "## Ignore Dependency Comparison"

For examples see [portal-impl/src/portal.properties](https://github.com/liferay/liferay-portal/blob/master/portal-impl/src/portal.properties)